config/routes.rb

```diff
@@ -71,6 +71,7 @@ def set_up_flipper
     end
     resources :questions
     resources :broadcast_announcements
+    resources :ndbn_members, only: %i(index create)
```

Comment:
`create` doesn't match with REST semantics - it implies you're creating a single member, not doing a bulk upload. I'd use a different route name here (e.g. `upload_csv`).